vmdb/app/helpers/container_node_helper/textual_summary.rb

```diff
@@ -40,4 +41,34 @@ def textual_ems
     end
     h
   end
+
+  def textual_num_cpu_cores
+    {:label => "Number of CPU Cores",
+     :value => @record.hardware.nil? ? "N/A" : @record.hardware.logical_cpus}
+  end
+
+  def textual_memory
+    if @record.hardware.nil? || !@record.hardware.memory_cpu.kind_of?(Numeric)
```

Comment:
This can be simplified to `if @record.try(:hardware).try(:memory_cpu)`.  The column is an integer, so there is no way this can be anything but nil or Numeric and this handles all cases.  Not necessary for this PR, but perhaps in a followup cleanup PR.